types/parse-diff/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for parse-diff 0.4
+// Project: https://github.com/sergeyt/parse-diff
+// Definitions by:  Lorens León <https://github.com/leon19>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ *  Parses unified diff
+ *  http://www.gnu.org/software/diffutils/manual/diffutils.html#Unified-Format
+ * @param input Diff string to get parsed
+ */
+declare function parseDiff(input?: string | null): parseDiff.File[];
```

Comment:
Whats the reason for having input typed as ```input?: string | null``` instead of ```input?: string```. The dopcumentation you provided doesnt mention it anywhere

Reply:
There is a [test in the original package](https://github.com/sergeyt/parse-diff/blob/master/test/tests.coffee#L5) for `null` inputs, since `null` is different than being optional I thought the definitions should match the original function inputs.

BTW, this pull request is no longer needed, a pull request that adds definitions to the package itself was accepted, feel free to delete this one.